oops/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
We should add `.vscode` to `.gitignore` and remove this file since we will all have our own preferences.